csi/controller.go

```diff
@@ -614,14 +613,17 @@ func (s *OsdCsiServer) CreateVolume(
 		}
 		newVolumeId = createResp.VolumeId
 	} else {
-		clonedMetadata := getClonedPVCMetadata(locator)
+		labels := locator.GetVolumeLabels()
 		if spec.GetFADAPodName() != "" {
-			clonedMetadata[sdk.FADAPodLabelKey] = spec.GetFADAPodName()
+			labels[api.SpecPurePodName] = spec.GetFADAPodName()
 		}
+		delete(labels, intreePvcNameKey)
```

Comment:
Why do we need to delete these?

Reply:
My thought was that the PVC name and namespace would no longer be applicable on the clone (and in unit tests when I added this, the labels were empty). However, let me sanity check what happens when I remove this and try a clone on a PVC volume, I'll see if it gets the label of the source or an empty string.

Reply:
Actually this doesn't seem necessary - while I'll have to update unit tests, the end functionality is fine without these delete statements.

Reply:
I thought these would be the name and namespace of the PVC (clone) that is getting created and not of the parent volume. Isn't that the case?